plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/BasicSQLDialect.java

```diff
@@ -213,6 +213,9 @@ public DBPIdentifierCase storesQuotedCase() {
     @NotNull
     @Override
     public String escapeString(String string) {
+        if (isEscapeBackslash()) {
+            return string.replace("'", "''").replace("\\", "\\\\");

```

Comment:
Do not copy-paste.

if (isEscapeBackslash()) {
            string = string.replace("\\", "\\\\");
}